tls/extensions/s2n_client_supported_versions.c

```diff
@@ -13,16 +13,16 @@
  * permissions and limitations under the License.
  */
```

Comment:
Did you mean to change the permissions on this file?

Reply:
Yes. I don't think our C files should be executable. Perhaps it should be a different PR?

Reply:
https://github.com/aws/s2n-tls/pull/3635#discussion_r1028582957 I commented this on a couple. I would say different PR-- those changes aren't obvious, particularly mixed into all the formatting changes. Let's make them deliberately and transparently.

Reply:
Done.